File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        while (s.back() == ' ') s = s.substr(0, s.size()-1);

        int i = 0;
        for (int j = s.size()-1; j>=0 && s[j] != ' '; j--){
            i++;
        }
        return i;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        temp = ""
        ans = ""
        for i in s:
            if i != " ":
                temp += i
            elif len(temp) > 0:
                ans = temp
                temp = ""
        if len(temp) > 0:
            ans = temp
        return len(ans)